README.md

```diff
@@ -0,0 +1,130 @@
+<div align="center">
+  <h3><b>Hello-Rails-Backend README</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    <!-- - [Key Features](#key-features) -->
+  <!-- - [🚀 Live Demo](#live-demo) -->
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  <!-- - [Run tests](#run-tests) -->
+  <!-- - [Deployment](#triangular_flag_on_post-deployment) -->
+- [👥 Authors](#authors)
+<!-- - [🔭 Future Features](#future-features) -->
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+<!-- - [🙏 Acknowledgements](#acknowledgements) -->
+<!-- - [❓ FAQ (OPTIONAL)](#faq) -->
```

Comment:
- [ ] Kindly uncomment this and I highly recommend to add Future Features and Acknowledgements sections to your README since they are very important to let anyone know what is the next step in your project 🤩💯✨⭐💪🏻